ansible/roles/test/files/ptftests/copp_tests.py

```diff
@@ -31,15 +31,15 @@ class ControlPlaneBaseTest(BaseTest):
     PPS_LIMIT_MIN = PPS_LIMIT * 0.9
     PPS_LIMIT_MAX = PPS_LIMIT * 1.1
     NO_POLICER_LIMIT = PPS_LIMIT * 1.4
-    PKT_TX_COUNT = 100000
-    PKT_RX_LIMIT = PKT_TX_COUNT * 0.90
     TASK_TIMEOUT = 300 # Wait up to 5 minutes for tasks to complete
 
     def __init__(self):
         BaseTest.__init__(self)
         self.log_fp = open('/tmp/copp.log', 'a')
         test_params = testutils.test_params_get()
         self.verbose = 'verbose' in test_params and test_params['verbose']
+        self.pkt_tx_count = 'pkt_tx_count' in test_params and test_params['pkt_tx_count']
```

Comment:
self.pkt_tx_count would be False if it's not in test_params.
Also I'd avoid save values with different types in one variable. (False or 100000)